### ISSUE TITLE
Remove EXCLUDE: syntax

### ISSUE THREAD

**Issue by bjourne** [factor/factor#1839]
This syntax is for using a vocab, but excluding some words. I don't think we need it anymore because if the same word name is imported from multiple vocabs, you can now always resolve the ambiguity by using the `vocab-name:word-name` form.

**Comment by catb0t** [comment]
Is `EXCLUDE:` an optimisation as well? The whole vocab still has to be loaded, right?

**Comment by mrjbq7** [comment]
Not an optimization, just helps the parser work around words that might be named the same from two vocabularies.

**Comment by mrjbq7** [comment]
So, +1 for removal.

**Comment by bjourne** [comment]
It's the same with the `QUALIFIED:` syntax too, isn't it? It is used in many more places though so it is probably harder to get  rid of.

**Comment by mrjbq7** [comment]
Right, I thought about that when I implemented "auto-qualified", but thought maybe "foo.bar.baz:qux" might be too verbose for some deep package names, so having a shorter way might be useful to some people. Not sure.

**Comment by erg** [comment]
Also consider removing ``RENAME:``.
